### PR TITLE
Consolidate code in CorrectZoneContaminants

### PR DESCRIPTION
**Pull request by lgu1234**
Pull request overview
---------------------
A section with ZoneMassFlowRate < = 0 was removed in CorrectZoneContaminants to make items in both CO2 and contaminant balance calculations are consistent with ones in energy and moisture balance calculations.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
   - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
   - NewFeature: This pull request includes code to add a new feature to EnergyPlus
   - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
   - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [x] Code style (parentheses padding, variable names)
 - [x] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [x] CI status: all green or justified
 - [x] Performance: CI Linux results include performance check -- verify this
 - [x] Unit Test(s)

### CODE REVIEW

**Comment by mjwitte** [comment]
@lgu1234 Please merge in develop and resolve the conflict on this branch.

**Comment by mjwitte** [comment]
@lgu1234 Or maybe you should wait with this one until I merge #6462?

**Comment by lgu1234** [comment]
@mjwitte  I will wait after you merger #6462,

**Comment by lgu1234** [comment]
@mjwitte I solved conflicts using VS 2015 and hope CI will give me green.

File: tst/EnergyPlus/unit/ZoneContaminantPredictorCorrector.unit.cc [hunk 1]
```diff
@@ -261,17 +262,17 @@ TEST_F( EnergyPlusFixture, ZoneContaminantPredictorCorrector_AddMDotOATest )
...
+	ZoneEquipConfig( 1 ).ExhaustNode.deallocate( );
+	ZoneEquipConfig.deallocate( );
+	Node.deallocate( );
+	Zone.deallocate( );
+	ZoneEqSizing.deallocate( );
+	OAMFL.deallocate( );
+	VAMFL.deallocate( );
+	EAMFL.deallocate( );
+	CTMFL.deallocate( );
+	MDotOA.deallocate( );
```
Comment (mjwitte) [inline tst/EnergyPlus/unit/ZoneContaminantPredictorCorrector.unit.cc]: @lgu1234 The deallocates here and in the next unit test are not needed. A quick look says they are all covered in their respective clear_state functions which all get called by `EnergyPlusFixture:: clear_all_states`.

Comment (lgu1234) [reply tst/EnergyPlus/unit/ZoneContaminantPredictorCorrector.unit.cc]: @mjwitte I am going to delete all and will commit it.

**Pull request merged**